conf/base.config

```
@@ -113,4 +113,15 @@ process {
         cpus   = { log_increase_cpus(2, 2*task.attempt, 1, 2) }
         time   = { check_max( 2.h * task.attempt, 'time') }
     }
+
+    withName: PRETEXTMAP {
+        cpus    = { 4        * task.attempt }
+        memory  = { 3.GB     * task.attempt }
+        time   =  { 1.h      * ( ( fasta.size() < 4e9 ? 24 : 48 ) * Math.ceil( task.attempt * 1 ) ) }
```

This is the same, right ?
```suggestion
        time   =  { 1.h      * ( fasta.size() < 4e9 ? 24 : 48 ) * task.attempt }
```

---

Done